Mastodon/Scene/HashtagTimeline/HashtagTimelineViewController.swift

```diff
@@ -42,7 +42,7 @@ final class HashtagTimelineViewController: UIViewController, NeedsDependency, Me
         
     let composeBarButtonItem: UIBarButtonItem = {
         let barButtonItem = UIBarButtonItem()
-        barButtonItem.image = Asset.ObjectsAndTools.squareAndPencil.image.withRenderingMode(.alwaysTemplate)
```

Comment:
It looks like this asset is only used in `DataSourceProvider+StatusTableViewCellDelegate.swift` so if you update that usage too it can be removed entirely to slim down the binary a bit